### ISSUE TITLE
SmartCity-Accessibility Server Side #438 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#282]
Build 'SmartCity-Accessibility Server Side' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [yaelAmitay] c7f89ba2de8c1e20e1e879c94e61835bb944756b - add sha to login add class token #253

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/438/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
